invenio_communities/assets/semantic-ui/js/invenio_communities/members/components/bulk_actions/SelectedMembers.js

```diff
@@ -22,31 +22,32 @@ export class SelectedMembers extends Component {
 
   render() {
     const { selectedMembers, displayingGroups } = this.props;
+    console.log(selectedMembers);
 
     return !_isEmpty(selectedMembers) ? (
       <>
-        <Header as="h3" size="small">
+        <Header as="h2" size="small">
           {i18next.t(displayingGroups ? "Selected groups" : "Selected members")}
         </Header>
-        <Segment
-          className="selected-members-header mb-20 mr-20"
-          role="list"
-        >
+        <Segment className="selected-members-header mb-20 mr-20">
```

Comment:
did you format the code with prettier ?

Reply:
I did, I made sure to run it manually now and still it doesn't change. What is the problem?